_includes/resume.html

```diff
@@ -0,0 +1,15 @@
+<h2>Resume/CV</h2>
+<div class="card-container">
+    {% for item in site.data.resume %}
+    <div class="card">
+      <div class="icon">
+        <img src="./assets/img/resume/cv.png">
+      </div>
+      <div class="info">
+        <p class="title"><a href="{{ item.link }}"target="_blank" rel="noopener noreferrer">{{ item.position }}</a></p>
```

Comment:
Hey @oshinsaini it would be nice to add an `a` tag to click on so we can go to the resume link and please give it an `id` of `#resume-link` so that I could get the link and integrate it with AlanAI so we can just ask him to download the resume by saying _download the resume_

Reply:
Hey @rosekamallove,  so you want a button or something next to the resume/cv for downloading purpose? Can you elaborate a bit. Thank you!

Reply:
If you are available let's get on a 1-1 and discuss

Reply:
Hey @rosekamalove, the resume already opens in a tab and allows the viewer to download the resume. Wouldn't it be redundant to add it twice?